spring-cloud-streams-samples/sink/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,6 @@
+---
+spring:
+  cloud:
+    config:
+      uri: http://localhost:8888
+      enabled: false
```

Comment:
Great Question!  In this case if the config server isn't present a benign error message is posted.  Just trying to give a clean out of the box experience for the user.  I can add instructions to the README to discuss enabling the cloud config.